edx_proctoring/__init__.py

```diff
@@ -4,5 +4,3 @@
 
 # Be sure to update the version number in edx_proctoring/package.json
 __version__ = '4.15.1'
```

Comment:
Let's bump the version as well. 
```suggestion
__version__ = '4.16.0'
```

Reply:
done with comment.